<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCAFacesDialog</class>
 <widget class="QDialog" name="PCAFacesDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>967</width>
    <height>475</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>820</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PCA Faces</string>
  </property>
  <widget class="QWidget" name="imageWidget" native="true">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>122</y>
     <width>256</width>
     <height>231</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>256</width>
     <height>200</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>500</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QWidget" name="dataWidget" native="true">
   <property name="geometry">
    <rect>
     <x>401</x>
     <y>52</y>
     <width>380</width>
     <height>340</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>337</x>
     <y>215</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinE1">
   <property name="geometry">
    <rect>
     <x>825</x>
     <y>352</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinE2">
   <property name="geometry">
    <rect>
     <x>885</x>
     <y>352</y>
     <width>41</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="value">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="spinE1label">
   <property name="geometry">
    <rect>
     <x>821</x>
     <y>322</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Eigen Vectors</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="spinE2label">
   <property name="geometry">
    <rect>
     <x>872</x>
     <y>352</y>
     <width>16</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>x</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>821</x>
     <y>392</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>82</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>or drag and drop:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>411</x>
     <y>402</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>left/right-click: change class label</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="eigenCountSpin">
   <property name="geometry">
    <rect>
     <x>851</x>
     <y>287</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="maximum">
    <number>99</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>801</x>
     <y>262</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Eigen Vectors to keep</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>411</x>
     <y>422</y>
     <width>166</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ctrl+click: remove sample</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>22</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="webcamButton">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>From Webcam</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="loadImgButton">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Load Image</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clipboardButton">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Loads images which have been saved via mouse left-click&lt;/p&gt;&lt;p&gt;save.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>From Clipboard</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>372</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>211</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select a region from this image 
        (click to select all)</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>821</x>
     <y>42</y>
     <width>102</width>
     <height>208</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="loadDatasetButton">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Load Dataset</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveDatasetButton">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Save Dataset</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clearButton">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Clear Dataset</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="samplesLabel">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Samples: 0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="eigenButton">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Eigenvectors</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>imageWidget</zorder>
  <zorder>dataWidget</zorder>
  <zorder>loadImgButton</zorder>
  <zorder>addButton</zorder>
  <zorder>samplesLabel</zorder>
  <zorder>clipboardButton</zorder>
  <zorder>spinE1</zorder>
  <zorder>spinE2</zorder>
  <zorder>spinE1label</zorder>
  <zorder>spinE2label</zorder>
  <zorder>closeButton</zorder>
  <zorder>clearButton</zorder>
  <zorder>loadDatasetButton</zorder>
  <zorder>saveDatasetButton</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>webcamButton</zorder>
  <zorder>eigenButton</zorder>
  <zorder>eigenCountSpin</zorder>
  <zorder>label_8</zorder>
  <zorder>label_7</zorder>
  <zorder></zorder>
  <zorder></zorder>
  <zorder>label_2</zorder>
  <zorder>imageWidget</zorder>
  <zorder>horizontalSpacer_2</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
